merkle/benches/crypto_sha512.rs

```diff
@@ -15,11 +15,10 @@ use crypto::sha2::Sha512;
 use hash512::Hash512;
 use merkletree::hash::{Algorithm, Hashable};
 use merkletree::store::{DiskStore, VecStore};
-use merkletree::merkle::{FromIndexedParallelIterator, MerkleTree};
+use merkletree::merkle::{FromIndexedParallelIterator, FromIteratorWithConfig, MerkleTree};
```

Comment:
Passing `None` to all of these constructors is a bit unfortunate.

How about we keep `FromIterator` which does not accept a `Config`, and require the config for `FromIteratorWithConfig`?